docs/index.rst

```diff
@@ -23,6 +23,23 @@ sunkit-image
           generated/gallery/index
           changelog
 
+Mission Statement
+=================
+
+The goal of the ``sunkit-image`` library is to provide access to image processing routines that are:
+
+1. Focused on being applied to solar physics data.
+2. Are published in the literature.
```

Comment:
What about that are in-prep? I like the idea of people writing their algorithms in sunkit-image before they publish so they can say that's where it will be in the paper? Or does this open us up to "oh we are going to publish this eventually"?

Reply:
How about if they can prove its in prep?

Reply:
How about we state it must be in prep and if we are skeptical for any reason we can only merge the PR when it's close to publication?

Reply:
Sure.